styleguide/source/assets/js/nav-primary.js

```diff
@@ -115,6 +115,52 @@ jQuery.noConflict();
       }
     });
 
+    // hover functionality @tablet and greater
+    $('.nav-primary_list-item').hover(
+      // Over
+      function () {
+        if ($(window).width() > 740) {
+          $(this).children('.nav-primary_list-item_title').children('.link-primary-nav').blur();
```

Comment:
I'd like to DRY-ify this if possible in the time that we have. The hover/click open/close seems to be duplicating a lot of lines, and that's not great. It seems like we should be able to add functions for `menuOpen(tab)` and `menuClose(tab)` and then use those for the close behavior I mentioned in the other comment.